### PR TITLE
Remove duplicate package list from install task

### PR DESCRIPTION
**Pull request by zmc**
Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
test this please

File: teuthology/nuke.py [hunk 1]
```diff
@@ -258,9 +258,9 @@ def remove_installed_packages(ctx):
     install_task.remove_packages(
         ctx,
         conf,
-        {"deb": install_task.deb_packages['ceph'] +
+        {"deb": install_task.PACKAGES['ceph']['deb'] +
          ['salt-common', 'salt-minion', 'calamari-server'],
```
Comment (andrewschoen) [inline teuthology/nuke.py]: Can these be part of PACKAGES as well, seems weird to tack them on here.  Maybe a common_packages list of some sort that gets added to both the deb and rpm lists?

**Comment by andrewschoen** [comment]
It looks like rpm_packages and deb_packages are being used in nuke.remove_installed_packages

https://github.com/ceph/teuthology/blob/master/teuthology/nuke.py#L255

**Comment by zmc** [comment]
They are, in master. Not in my branch...

File: teuthology/nuke.py (hunk 1 above)
Comment (zmc) [inline teuthology/nuke.py]: I agree it's weird, but I don't think we want to install them whenever we use the install task. Not trying to fix that problem here.

Comment (andrewschoen) [inline teuthology/nuke.py]: Fair enough

File: teuthology/task/install.py [hunk 2]
```diff
@@ -776,10 +740,9 @@ def install(ctx, config):
 
     project = config.get('project', 'ceph')
 
-    global deb_packages
-    global rpm_packages
-    debs = deb_packages.get(project, [])
-    rpm = rpm_packages.get(project, [])
+    global PACKAGES
```
Comment (andrewschoen) [inline teuthology/task/install.py]: This doesn't need to be global does it?

Comment (zmc) [inline teuthology/task/install.py]: I don't think so, but I was trying to make as little impact as possible with the change.

Comment (andrewschoen) [inline teuthology/task/install.py]: I can appreciate that concern.  At some point we should start removing cruft like this though.

FWIW, I see PACKAGES being used in install.upgrade_common, but it doesn't seem to depend on  any mutation that happens in install.install caused by it being global.

**Comment by andrewschoen** [comment]
Thanks for removing that global, ship it.

**Pull request merged**